cmd/apprepository-controller/main.go

```diff
@@ -75,4 +78,7 @@ func init() {
 	flag.StringVar(&kubeconfig, "kubeconfig", "", "Path to a kubeconfig. Only required if out-of-cluster.")
 	flag.StringVar(&masterURL, "master", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
 	flag.StringVar(&repoSyncImage, "repo-sync-image", "kubeapps/chart-repo:latest", "container repo/image to use in CronJobs")
+	flag.StringVar(&namespace, "namespace", "kubeapps", "Namespace to discover AppRepository resources")
```

Comment:
May I ask why these changes are related to the creation of a chart? Couldn't this have been done in another previous patch?

Just trying to understand the dynamics :)

Reply:
This was needed because previously apprepository-controller was hardcoded to use the `kubeapps` namespace which was fine as `kubeapps up` didn't make that configurable. The chart can be installed in any namespace though (although we could hardcode it, but bad practice), so this is needed for that to work.